### PR TITLE
[vmm]: support configure log_level in the config file

### LINKED ISSUE #35: Support configure the vmm-sandboxer's log level in the config file
### What would you like to be added?

Add a new config field under '[sandbox]' module in the config toml file:
```toml
[sandbox]
# Set the vmm-sandboxer process's log level
# The valid log_level value is: "error", "warn", "info", "debug", "trace"
# Default log level is "info"
log_level = "info"
```

### Why is this needed?

Currently, vmm-sandboxer's log_level is specified by `RUST_LOG` env when starting the vmm-sandboxer process.
But it is not a good way to configure the vmm-sandboxer's log_level especially when running vmm-sandboxer process as a systemd service.
So we want to add a more flexible method to configure the vmm-sandboxer's log level in the config file.

### PR DESCRIPTION
**Pull request by flyflypeng**
fix #35

reason: Add 'sandbox.log_level' field in the config file to support configure the vmm-sandboxer process log level.

### CODE REVIEW

File: vmm/sandbox/src/main.rs [hunk 1]
```diff
@@ -102,9 +102,8 @@ pub const FS_SHARE_PATH: &str = "shared_fs";
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
-    env_logger::Builder::from_default_env()
-        .format_timestamp_micros()
-        .init();
+    let mut builder = env_logger::Builder::from_default_env();
```
Comment (Burning1020) [inline vmm/sandbox/src/main.rs:105]: How about this?
```Rust
        let mut builder = env_logger::Builder::from_default_env().format_timestamp_micros();

...
        if !config.sandbox.log_level.is_empty() 
            let log_level = log::LevelFilter::from_str(config.sandbox.log_level.as_str())?;
            builder.filter_level(log_level);
        }
        builder.init();
...

```

Comment (flyflypeng) [reply vmm/sandbox/src/main.rs:105]: Nice! More clean code! 👍🏼

File: vmm/sandbox/src/main.rs [hunk 2]
```diff
@@ -145,6 +144,19 @@ async fn main() -> anyhow::Result<()> {
         } else {
             panic!("config file {} not exist", config_path);
         };
+
+        // If 'log_level' field isn't set in the config file, keep the log level from the default env
+        // Otherwise, set the log level configured in the config file
+        if config.sandbox.log_level.is_empty() {
```
[the remark this answers is not in the document]
Comment (flyflypeng) [reply vmm/sandbox/src/main.rs]: This is a good idea, but I think it's not convenient for users to do the configuration in more than one configuration file. If wasm/quark/vmm sandbox has more common abstract configuration items in the future, I think a separate configuration file for all sandbox types can be created to store these configuration items.

File: vmm/sandbox/src/main.rs [hunk 3]
```diff
@@ -100,11 +100,14 @@ pub const NAMESPACE_CGROUP: &str = "cgroup";
 
 pub const FS_SHARE_PATH: &str = "shared_fs";
 
+pub const CONFIG_STRATOVIRT_PATH: &str = "/var/lib/kuasar/config_stratovirt.toml";
```
Comment (abel-von) [inline vmm/sandbox/src/main.rs:103]: It seems the constant defined here has no use anywhere?

File: vmm/sandbox/src/main.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (abel-von) [reply vmm/sandbox/src/main.rs]: Sorry, maybe I did not make it clear of what I mean, the config parse logic is almost the same for all the hypervisors, maybe we can eliminate the duplicate codes.  I also think it is not the time to make a global config file.

[the remark this answers is not in the document]
Comment (flyflypeng) [reply vmm/sandbox/src/main.rs]: Ok, I have refactored the main function to remove the repeated code.

File: vmm/sandbox/src/main.rs (hunk 3 above)
Comment (flyflypeng) [reply vmm/sandbox/src/main.rs:104]: Fix it!

**Comment by Vanient** [comment]
LGTM

**Pull request merged**